src/server/cronjobs.rs

```diff
@@ -0,0 +1,61 @@
+use crate::actions::{Action, ActionsCtx, UpdateLists};
+use crate::prelude::*;
+use crate::server::Data;
+use crate::utils;
+use std::sync::Arc;
+use std::thread;
+use std::time::Duration;
+
+const DAY: Duration = Duration::from_secs(60 * 60 * 24);
+// The tuple is composed of:
+// - job name
+// - interval between executions
+// - function to execute at specified intervals
+type JobDescription = (&'static str, Duration, fn(Arc<Data>) -> Fallible<()>);
```

Comment:
Could this be a struct instead? That way the description of the fields would be in the struct field names instead of a comment.